### ISSUE TITLE
McCode: libraries are put in libs - instead of lib

### ISSUE THREAD

**Issue by ebknudsen** [McStasMcXtrace/McCode#1084]
We should prob. use the more standard terminology and use lib instead. That way we would avoid having both "lib" and "libs".

**Comment by willend** [comment]
- the same goes for share -> include ?

**Comment by farhi** [comment]
In fact `libs` contains some `contrib` code (cif2hkl, mcpl).
And share contains both C and H files. Perhaps `share` is then OK ? Include usually means only H files.

**Comment by willend** [comment]
@farhi share probably makes sense, yes. During the last Hackathon I had a chat with @tkittel about the installation "layout" and the different types of comps/instrs/code/data etc., we didn't reach an exhaustive conclusion.

For the next set of releases I think we should leave "as is" even though it is a bit of a mess - then we can think of a better and more standard layout of everything afterwards.

**Comment by tkittel** [comment]
Yes, after the next release is out I recommend we try to stick to platform defaults (i.e. using the CMake GNU standards when on unix). So bin/ for binaries,  include/ for down-stream includable headers, lib/lib64/whatever for libs, share/McStas/<subdirs> for everything else.

There will be a bunch of work to make sure everything is configurable, but maybe not too much :-)

**Comment by willend** [comment]
Better solution in place now / via conda-forge deployment
